[k9][CSM][Jira] Update docs (#20619)
DataDog/documentation: content/en/security/cloud_security_management/guide/jira.md

Use the [Jira integration][1] to create Jira issues for resources that are impacted by a Cloud Security Management (CSM) security issue. Jira for Cloud Security Management is available for [CSM Misconfigurations][3] and [CSM Identity Risks][4].

> **Note**: To create Jira issues, you must have the `security_monitoring_findings_write` permission. See [Role Based Access Control][2] for more information about Datadog's default roles and granular role-based access control permissions available for CSM. We currently support creating one ticket per finding.

Reviewer: ```suggestion
**Notes**: 
- To create Jira issues, you must have the `security_monitoring_findings_write` permission. See [Role Based Access Control][2] for more information about Datadog's default roles and granular role-based access control permissions available for CSM. 
- At this time, you can create only one Jira issue per finding.
```

1. On the [Misconfigurations Explorer][1], select a misconfiguration.

> 2. Under **Resources Impacted**, select one or more resources.

Reviewer: ```suggestion
2. Under **Resources Impacted**, select one or more findings.
```

> 3. On the **Actions** dropdown menu that appears on top, select **Create Jira Issue**.

Author: I'm adding this detail because I think it's very easy to confuse these two buttons
![Screenshot 2023-11-14 at 16 52 15](https://github.com/DataDog/documentation/assets/122295290/d4b5258d-07ec-4208-a907-a709f4c1a010)

[2]: /account_management/rbac/permissions/#cloud-security-platform
[3]: /security/misconfigurations/
[4]: /security/identity_risks/

> [5]: https://app.datadoghq.com/integrations/jira?search=jira

Author: auto-formatting adding empty-line at the bottom
